src/registry_schemas/schemas/order.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/order",
+    "type": "object",
+    "definitions": {},
+    "title": "Order Schema",
+    "required": [
+        "orderDetails"
+    ],
+    "properties": {
+        "fileNumber": {
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 20,
+            "description": "The court assigns each court order a unique file number up to 20 characters in length."
+        },
+        "orderDate": {
+            "type": "string",
+            "description": "The date and time of the order.",
+            "format": "date-time",
+            "examples": ["1970-01-01T00:00:00+00:00"]
+        },
+        "effectOfOrder": {
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 500,
+            "description": "planOfArrangement"
+        },
+        "orderDetails": {
+            "type": "string",
+            "minLength": 1,
+            "maxLength": 2000,
+            "description": "A brief note to explain the purpose of the Order."
+        }
+    }
```

Comment:
Should `courtOrder` be in here? If not, where is it?

Reply:
Answered by comment above.

Reply:
No. This include those prop's as same as registrars order, registrars notation